scripts/plugins/storage/sqlite/upgrade/75.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE fledge.streams ADD COLUMN last_objects JSON  NOT NULL DEFAULT '{"Readings":0,"Stats":0,"Audit":0}';
```

Comment:
As with other upgrader scripts, this does not preserve the location in the readings